src/Geralt/Crypto/Argon2id.cs

```diff
@@ -43,6 +57,14 @@ public static bool VerifyHash(ReadOnlySpan<byte> hash, ReadOnlySpan<byte> passwo
         return crypto_pwhash_str_verify(hash, password, (ulong)password.Length) == 0;
     }
 
+    public static bool VerifyHash(string hash, ReadOnlySpan<byte> password)
+    {
+        Validation.NotNull(nameof(hash), hash);
```

Comment:
Please change this to `Validation.NotNullOrEmpty()`.

Reply:
Actually, it's probably better to do `Validation.NotNull()` and then `Validation.SizeBetween()` with the `MinHashSize` and `MaxHashSize` constants.